apollo/smoothie.py

```diff
@@ -58,70 +24,182 @@ def connect(self):
                 loop=self.loop
             )
         except OSError as e:
-            logger.log("foo")
+            logger.error("Did not connect")
```

Comment:
Probably make this say,

"Failed to establish connection with the SmoothieBoard"